frontend/src/components/table/TableItem.js

```diff
@@ -9,18 +8,22 @@ import {
   prepareWidgetData,
   isEditableOnDemand,
   isCellEditable,
+  tableRowPropTypes,
   getCellWidgetData,
+  getDescription,
+  getTdValue,
+  nestedSelect,
 } from '../../utils/tableHelpers';
 
 import TableCell from './TableCell';
 import WithMobileDoubleTap from '../WithMobileDoubleTap';
 
 /**
  * @file Class based component.
- * @module TableItem
+ * @module TableRow
  * @extends PureComponent
  */
-class TableItem extends PureComponent {
+class TableRow extends PureComponent {
```

Comment:
Pls rename the file to match the component name -> TableRow . Also this implies updates in all the files that do import that.